tools/platforms/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+package(
+    default_visibility = ["//visibility:public"],
```

Comment:
Please don't use `default_visibility`. Just add `visibility` to individual targets. Over time this file may grow, and may gain targets that should not be public.